spec/system/projects/portfoliorrr_profiles/user_views_portfoliorrr_profile_details_spec.rb

```diff
@@ -73,16 +66,59 @@
       within '#project-navbar' do
         click_on 'Procurar usuários'
       end
-      click_on 'Rodolfo'
 
-      expect(page).to have_content 'Projeto Top - Recrutamento de colaboradores'
+      expect(page).to have_link 'Rodolfo', href: project_portfoliorrr_profile_path(project, rodolfo_profile.id)
+    end
+
+    it 'com sucesso' do
+      user = create :user, cpf: '149.759.780-32'
+      project = create :project, user:, title: 'Projeto Top'
+      project.user_roles.find_by(user:).update(role: :leader)
```

Comment:
Qual o contexto desse update?

Reply:
Se for pelo bug de duplicar, só acontece se você inserir manualmente o líder como contribuidor

Reply:
Eu e o @fredericomozzato deixamos assim porque a gente criou o teste com base no anterior que tinha no arquivo.

O teste anterior foi feito por mim e pelo @mateuscavedini. A gente fez assim pra ficar claro que o usuário é líder. No caso, se remover a linha do `update`, funcionaria igual, ja que por padrão o `user_role` é criado com o `role` de líder.

De verdade, não sei qual jeito seria melhor, com o update ou não :/

Reply:
Ah sim, acho que trocar o nome da variável de user pra leader e omitir o user_role funciona também, mas é mais uma escolha pessoal mesmo